typings/axe-core/axe-core.d.ts

```diff
@@ -0,0 +1,152 @@
+// Type definitions for axe-core 2.0.5
+// Project: https://github.com/dequelabs/axe-core
+// Definitions by: Marcy Sutton <https://github.com/marcysutton>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface ElementContext {
+	node?: Object,
+	selector?: string,
+	include?: any[],
+	exclude?: any[]
+}
+interface RunOnly {
+	type: string, // "rule" | "rules" | "tag" | "tags"
+	value?: {
+		include?: string[],
+		exclude?: string[]
+	}
+	values?: string[] // "wcag2a" | "wcag2aa" | "section508" | "best-practice"
+}
+interface AxeResults {
+	url: string,
+	timestamp: string,
+	passes: Pass[],
+	violations: Violation[]
+}
+interface Pass {
+	description: string,
+	help: string,
+	helpUrl: string,
+	id: string,
+	impact: string, // "minor" | "moderate" | "serious" | "critical" | "null"
```

Comment:
Yep, good catch. I want to abstract these strings into a common place since they're used multiple times – same deal with the tag values above. Any tips on how to do that without depending on Typescript 2.0? I'd like for it to be usable in either version if possible...